dmriqcpy/template/libs/js/scripts.js

```diff
@@ -801,8 +797,8 @@ function save_qc(){
     }
     if (person != null){
         var currentdate = new Date();
-        var datetime = currentdate.getDate() + "/"
-                        + (currentdate.getMonth()+1)  + "/"
+        var datetime = ("0" + currentdate.getDate()).slice(-2) + "/"
```

Comment:
You could use `String(currentdate.getDate()).padStart(2, "0")` choose the one you found cleaner.